app/api/routers/events.py

```diff
@@ -272,3 +299,81 @@ def list_all_ibet_security_token_escrow_event_logs(
         **SuccessResponse.use().dict(),
         "data": resp_json
     }
+
+
+# /Events/IbetSecurityTokenInterface/{token_address}
+@router.get(
+    "/IbetSecurityTokenInterface/{token_address}",
+    summary="List all IbetSecurityTokenInterface event logs",
+    operation_id="IbetSecurityTokenEscrowEvents",
+    response_model=GenericSuccessResponse[ListAllEventsResponse],
+    responses=get_routers_responses(
+        InvalidParameterError,
+        RequestBlockRangeLimitExceededError
+    )
+)
+def list_all_ibet_security_token_interface_event_logs(
```

Comment:
Added API for `IbetSecurityTokenInterface`.
You can use this to fetch lock/unlock events about given `token_address`.